### PR TITLE
feat: add tests.

### PR DESCRIPTION
**Pull request by IgorOvcharenko**

### CODE REVIEW

File: src/home.spec.ts [hunk 1]
```diff
@@ -24,3 +24,155 @@ test('Home Page has correct title and can open shop', async ({ page, context })
...
+
+
+test('When declining only analytics in cookie consent, Google Analytics should not be loaded 1', async ({ page }) => {
+  await page.goto('https://www.tapio.one/');
+
+  // save services
+  await page.locator("data-testid=uc-save-button").click();
+
+  //expect no google-analytics script
+  await expect(page.locator('script[src="https://www.google-analytics.com/analytics.js"]')).toHaveCount(0)
```
[the remark this answers is not in the document]
Comment (eimerreis) [reply src/home.spec.ts:165]: Try to figure out if this is possible with playwright and how

File: src/home.spec.ts [hunk 2]
```diff
@@ -24,3 +24,155 @@ test('Home Page has correct title and can open shop', async ({ page, context })
   // Expects the URL to contain intro.
   await expect(newPage).toHaveURL(/https:\/\/customerportal.tapio.one\/marketplace/);
 });
+
+
+test('Register Button in nav menu should lead to https://my.tapio.one/signup/landing?register=customer in a new tab', async ({ page, context }) => {
+  await page.goto('https://www.tapio.one/');
```
Comment (eimerreis) [inline src/home.spec.ts:30]: maybe we could use `beforeEach` for that

File: src/home.spec.ts [hunk 3]
```diff
@@ -24,3 +24,155 @@ test('Home Page has correct title and can open shop', async ({ page, context })
...
+
+  // accept cookies 
+  await page.locator("data-testid=uc-accept-all-button").click();
```
Comment (eimerreis) [inline src/home.spec.ts:33]: read about the `Page Object Model` in the playwright docs. I think we could use it for the cookie consent

**Comment by IgorOvcharenko** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by eimerreis** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**